docs/_source/components-overview/pop_syn/population_params.rst

```diff
@@ -674,6 +674,14 @@ It also contains which sampling distributions to use for the initial conditions
     - | In units of solar metallicity. Supported values ``[2., 1., 0.45, 0.2, 0.1, 0.01, 0.001, 0.0001]``
     - ``[1.]``
 
+  * - ``error_checking_verbose``
+    - | If True, write all POSYDON errors to stderr at runtime
+    - ``False``
+
+  * - ``warnings_verbose``
+    - | If True, write all POSYDON warnings to stderr at runtime
+    - ``False``
```

Comment:
Those are good additions, shell we mention, that independent on this value we get the online record?